lib/node.js

```diff
@@ -820,6 +821,82 @@ class Node extends rclnodejs.ShadowNode {
     return subscription;
   }
 
+  /**
+   * Create a Subscription that returns an RxJS Observable.
+   * This allows using reactive programming patterns with ROS 2 messages.
+   *
+   * @param {function|string|object} typeClass - The ROS message class,
+   *      OR a string representing the message class, e.g. 'std_msgs/msg/String',
+   *      OR an object representing the message class, e.g. {package: 'std_msgs', type: 'msg', name: 'String'}
+   * @param {string} topic - The name of the topic.
+   * @param {object} [options] - The options argument used to parameterize the subscription.
+   * @param {boolean} [options.enableTypedArray=true] - The topic will use TypedArray if necessary.
+   * @param {QoS} [options.qos=QoS.profileDefault] - ROS Middleware "quality of service" settings.
+   * @param {boolean} [options.isRaw=false] - The topic is serialized when true.
+   * @param {string} [options.serializationMode='default'] - Controls message serialization format.
+   * @param {object} [options.contentFilter] - The content-filter (if supported by RMW).
+   * @param {SubscriptionEventCallbacks} [eventCallbacks] - The event callbacks for the subscription.
+   * @return {ObservableSubscription} - An ObservableSubscription with an RxJS Observable.
+   */
+  createObservableSubscription(typeClass, topic, options, eventCallbacks) {
+    if (typeof typeClass === 'string' || typeof typeClass === 'object') {
+      typeClass = loader.loadInterface(typeClass);
+    }
+
+    options = this._validateOptions(options);
+
+    if (typeof typeClass !== 'function') {
+      throw new TypeValidationError('typeClass', typeClass, 'function', {
+        nodeName: this.name(),
+        entityType: 'subscription',
+      });
+    }
+    if (typeof topic !== 'string') {
+      throw new TypeValidationError('topic', topic, 'string', {
+        nodeName: this.name(),
+        entityType: 'subscription',
+      });
+    }
+    if (
+      eventCallbacks &&
+      !(eventCallbacks instanceof SubscriptionEventCallbacks)
+    ) {
+      throw new TypeValidationError(
+        'eventCallbacks',
+        eventCallbacks,
+        'SubscriptionEventCallbacks',
+        {
+          nodeName: this.name(),
+          entityType: 'subscription',
+          entityName: topic,
+        }
+      );
+    }
```

Comment:
The lines above are dup with
https://github.com/RobotWebTools/rclnodejs/blob/4a1b0db25c4d865bc78bd238bd9d325af6bfcbca/lib/node.js#L762